### ISSUE TITLE
Record telemetry for block cookies option.

### ISSUE THREAD

**Issue by mcarare** [mozilla-mobile/focus-android#6097]
The question is if we want to send it also along with other [info](https://github.com/mozilla-mobile/focus-android/blob/99643723c6a509dc98c7e24904b307d65bfd2ce0/app/src/main/java/org/mozilla/focus/telemetry/GleanMetricsService.kt#L94) at start-up, or just when the setting is changed.

@channingcarter Is this something that product and/or DS needs? Thank you!

**Comment by channingcarter** [comment]
Is this telemetry for TCP?

**Comment by mcarare** [comment]
@channingcarter For any cookie option, including TCP.

**Comment by channingcarter** [comment]
We can just send only when the setting is changed.

**Comment by lobontiumira** [comment]
Verified as fixed on the Focus Nightly build 97.0 (build 353610924 GV 97.0a1-20211226092042) with Google Pixel (Android 10).
[Glean dashboard](https://debug-ping-preview.firebaseapp.com/pings/mira-2112270001)

```
{
          "timestamp": 15956,
          "category": "privacy_settings",
          "name": "block_cookies_changed",
          "extra": {
            "is_enabled": "third_party_only"
          }
        }
```
